.travis.yml

```diff
@@ -0,0 +1,15 @@
+launguage: cpp
+os: osx
+addons:
+  homebrew:
+    packages:
+    - openmpi
+    - vtk
+    - swig
+    - cmake
+    - ninja
+install:
+  - pip3 install mpi4py
+script:
+  - git reset --hard ${TRAVIS_PULL_REQUEST_SHA}
```

Comment:
does Travis CI not do the clone & checkout the right branch/commit for us??

Reply:
It first does a merge of the PR branch with master, resulting in CDash showing some random SHA in its update column (one that doesn't match all the other rows).